packages/itwin/tree-widget/src/components/search-bar/SearchBar.tsx

```diff
@@ -117,21 +110,21 @@ SearchBarState
       <div className={classes}>
         {enableGrouping && <div className={contentClassName}>
           <ButtonGroup
+            // note: I have no idea why TS wants these attributes, since they're all optional..
+            // eslint-disable-next-line jsx-a11y/no-autofocus
+            autoFocus={undefined} nonce={undefined} content={undefined} rel={undefined} rev={undefined}
```

Comment:
I tried to do a clean build locally without these props and everything worked fine. 😕 Maybe there were some issues with dependencies on your machine?

Building with these props I get error that `ButtonGroup` does not have `autoFocus` property.

Reply:
yes, something was wrong with my set up - made a fresh install and now TS doesn't require those props anymore